### DISCUSSION TITLE
京东云pcdn没有流量

### DISCUSSION THREAD

**Discussion by a641764137** [coolsnowwolf/lede#9572] [R68S / 电犀牛专区]
京东云pcdn没有上传流量，已经开了fullcone nat但是没有一点效果，但是app检测已经是nat1了，流量跑不起来，望解决。

**Comment by ljlwdf** [comment]
已经知道原因了，关闭“SFE 加速”就好了。

**Reply by Bigbogai** [reply]
你好，还有没有其余的设置？
